### PR TITLE
Add instruction for SkyPilot

### PR DESCRIPTION
**Pull request by Michaelvll**
## Checklist

- [ ] Before submitting a PR for review, make sure it has passed verification in your local development environment **at least**.
- [ ] Ensure pre-commit `pre-commit run --all-files` or other linting tools are used to fix potential lint issues.
- [ ] Confirm that modifications are covered by complete unit tests. If not, please add more unit tests for correctness.
- [ ] Modify documentation as needed, such as docstrings or example tutorials.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -87,6 +87,45 @@ docker run --gpus all \
 1. Copy the [compose.yml](./docker/compose.yaml) to your local machine
 2. Execute the command `docker compose up -d` in your terminal.
 
+### Method 5: Run on Clouds with SkyPilot
```
Comment (concretevitamin) [inline README.md]:
```suggestion
### Method 5: Run on Kubernetes or Cloud VMs with SkyPilot
```

File: README.md [hunk 2]
```diff
@@ -87,6 +87,45 @@ docker run --gpus all \
...
+
+To deploy on any cloud or Kubernetes cluster, you can use [SkyPilot](https://github.com/skypilot-org/skypilot).
```
Comment (concretevitamin) [inline README.md]: L92:94: swap K8s and cloud?

File: README.md [hunk 3]
```diff
@@ -87,6 +87,45 @@ docker run --gpus all \
...
+
+1. Install SkyPilot and setup your cloud or Kubernetes cluster: see [SkyPilot's documentation](https://skypilot.readthedocs.io/en/latest/getting-started/installation.html).
```
Comment (concretevitamin) [inline README.md]:
```suggestion
1. Install SkyPilot and set up cloud VM or Kubernetes cluster access: see [SkyPilot's documentation](https://skypilot.readthedocs.io/en/latest/getting-started/installation.html).
```

File: README.md [hunk 4]
```diff
@@ -87,6 +87,45 @@ docker run --gpus all \
...
+  python3 -m sglang.launch_server \
+    --model-path meta-llama/Meta-Llama-3.1-8B-Instruct \
+    --host 0.0.0.0 \
+    --port 30000
+```
+
+</details>
+
+```bash
+# Deploy on any cloud or Kubernetes cluster, use --cloud <cloud> to select a specific cloud provider.
```
Comment (concretevitamin) [inline README.md]:
```suggestion
# Deploy on any cloud or Kubernetes cluster. Use --cloud <cloud> to select a specific cloud provider.
```

**Review by concretevitamin — APPROVED** [review]

File: README.md (hunk 1 above)
Comment (Michaelvll) [reply README.md]: "Run on Cloud VMs" feels like that we need to run on existing cloud VMs. How about just `Run on Kubernetes or Cloud with SkyPilot`?

Comment (concretevitamin) [reply README.md]: Sg, plural? `Run on Kubernetes or Clouds with SkyPilot`

**Pull request merged**